frame/elections-phragmen/src/lib.rs

```diff
@@ -602,6 +603,8 @@ pub mod pallet {
 		InvalidRenouncing,
 		/// Prediction regarding replacement after member removal is wrong.
 		InvalidReplacement,
+		/// Can't add another lock to the account.
+		TooManyLocks,
```

Comment:
Maybe like `Overflow` this can be put elsewhere.

Reply:
`TokenError` might be a good place. https://github.com/paritytech/substrate/blob/7ba4e4ced0c0ed4ce8ccba6b36f6d53d7de3db89/primitives/runtime/src/lib.rs#L531-L536

Reply:
Or we could have a more general `StorageError::OutOfBound` in `DispatchError`. Currently there are many repeating `TooManyXXX` errors for `BoundedVec` out of bound in pallets, they could be replaced.